### DISCUSSION TITLE
Mixed reality

### DISCUSSION THREAD

**Discussion by tturkmen** [USC-ACTLab/crazyswarm#441] [General]
I have 1 pyhsical cf with flow deck v2. I want to simulate 5 cfs. One of the cf is my physical cf. How can do this.

**Comment by whoenig** [comment]
There is no support for this out of the box. It mostly depends on what virtual environment you would like to use. If you just want to rely on the Crazyswarm "simulator" (which does not actually simulate any physics!), you might be able to mix physical and virtual Crazyflies, see https://github.com/USC-ACTLab/crazyswarm/blob/dfa154acc2026f56985463edcfb86dba7861af8d/ros_ws/src/crazyswarm/scripts/pycrazyswarm/crazyswarm.py#L42-L50.

**Reply by tturkmen** [reply]
thanks I try
